index.js

```diff
@@ -44,6 +44,20 @@ function findPackedTag(gitPath, sha) {
   }
 }
 
+function findPackedCommit(gitPath, refPath) {
```

Comment:
This looks very similar to the code for `findPackedTag`, can we extract some common helper?